src/build.py

```diff
@@ -620,17 +625,26 @@ def Fastcomp():
 
 def Emscripten():
   buildbot.Step('emscripten')
-  os.environ['EMSCRIPTEN'] = EMSCRIPTEN_SRC_DIR
-  os.environ['LLVM'] = os.path.join(INSTALL_DIR, 'fastcomp', 'bin')
-  os.environ['V8'] = os.path.join(INSTALL_DIR, 'bin', 'd8')
   em_config = os.path.join(INSTALL_DIR, 'emscripten_config')
+  emscripten_dir = os.path.join(INSTALL_DIR, 'emscripten')
+  os.environ['EM_CONFIG'] = em_config
+  Remove(emscripten_dir)
+  shutil.copytree(EMSCRIPTEN_SRC_DIR,
+                  emscripten_dir,
+                  symlinks=True,
+                  ignore=shutil.ignore_patterns('.git'))
```

Comment:
Emscripten's source checkout is also its distribution package. But we don't want to archive that big blob. Added a comment.